app/src/main/java/me/niccorder/news/data/DummyNewsRepository.java

```diff
@@ -112,21 +113,41 @@ private static Article createArticle(
             )
     };
 
+    /**
+     * A handler is a utility class used to communicate between a background thread and the main
+     * thread, or to run a task after a delay.
+     */
+    private final Handler handler;
+
+    public DummyNewsRepository(Handler handler) {
+        this.handler = handler;
+    }
+
     @NonNull
     @Override
     public List<Category> getAllCategories() {
         return Arrays.asList(
                 BUSINESS,
                 TECHNOLOGY,
                 POLITICS,
-                SPORTS
+                SPORTS,
+                WORLD
```

Comment:
Thanks for the bug-fix 💯 Next time, lets keep our PR's as to the point as possible to we can better track what changes are being made. For this PR it's fine though!

Thanks again 😄